scrapinghub/legacy.py

```diff
@@ -11,6 +11,8 @@
 import time
 import warnings
 
+from . import __version__
```

Comment:
hm, doesn't that lead to a circular import? you import from `legacy` in `__init__` and import from `__init__` in `legacy`